im-commons/src/main/java/com/wipro/ats/bdre/ResolvePath.java

```diff
@@ -26,8 +26,8 @@
  * Created by AS294216 on 1/5/2016.
  */
 public class ResolvePath {
-    private static Logger LOGGER = Logger.getLogger(ResolvePath.class);
-
+    private ResolvePath(){
+    }
 
     private static Pattern pattern = Pattern.compile("%(.)");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)